cf-java-logging-support-logback/src/main/java/com/sap/hcp/cf/logback/converter/CategoriesConverter.java

```diff
@@ -39,7 +42,7 @@ private void getMarkers(Marker marker, StringBuilder appendTo) {
             getMarkersRecursively(marker, ac);
             appendTo.append(ac.end().finish());
         } catch (IOException ex) {
-            LoggerFactory.getLogger(CategoriesConverter.class).error("conversion failed", ex);
+            LOG.error("conversion failed", ex);
```

Comment:
Can you please use the LoggerHolder in this class as well? Even without the comment, I think it can have the same issue on start up.

Reply:
Done with latest change.

Reply:
Thanks